### DISCUSSION TITLE
Are there no"live-logs" in Github CI's anymore ?

### DISCUSSION THREAD

**Discussion by raphaelscholle** [community/community#67238] [Actions] [answered]
### Select Topic Area

Bug

### Body

Hello! I've been relying on GitHub CI for debugging and building using GitHub runners quite extensively. Typically, I keep a dedicated screen open to monitor the build logs. However, I've noticed that for the past few days, these logs seem to have disappeared. I'm curious if this is an intentional change or if there might be an issue? Your insights would be greatly appreciated.

![image](https://github.com/community/community/assets/68374617/5a295da2-1335-4544-94bc-7dfaed22c8e1)

**Comment by matteocontrini** [comment]
Same issue here. Actually, it seems that real-time features often don't work at all. If I don't refresh the page I don't see the updated progress through the steps.

**Comment by raphaelscholle** [comment]
Looks like I'm not the only one

**Comment by raphaelscholle** [comment, marked as answer]
And now it started working again :) 

![image](https://github.com/community/community/assets/68374617/ef03be58-48f7-46fa-aab3-44137eaf8f59)

**Comment by matteocontrini** [comment]
Still not working here, but I guess it will at some point.

**Comment by hwittenborn** [comment]
It isn't working for me currently either. I suspected it might be do to my WiFi or something, but I'm experiencing the issue across multiple internet connections.

Looking at my browser's web page inspector (Firefox 117.0.1), it doesn't even look like there's requests being made to fetch new logs.
